spec/models/export_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+
+RSpec.describe Export do
+  describe ".enqueue_faxes" do
+    it "schedules jobs for signed, unfaxed application updated awhile ago" do
+      A_WHILE_AGO = 31.minutes.ago
```

Comment:
sorry, when I commented before I was not clear on my thinking about constants in this context.

What I meant to say that the fact that one is more than 30 mins (a while ago, in this app) and one is less than 30 mins (less than a while ago, in this app) is where the magic number comes in. 

That magic number is in `app/models/snap_application.rb`. Perhaps create a cleanup ticket to change that to a constant in a separate PR and then here, when we add that constant, we can say `CONSTANT.minutes + 1.minute` or  `CONSTANT.minutes - 5.minutes` -- does that make sense?